ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/widgets/editortab/EditorTabWidget.ui.xml

```diff
@@ -39,6 +41,13 @@
             color: tabTextColor;
             cursor: default;
             margin-bottom: 1px;
+
+            -webkit-transition: all 0.3s ease-in-out,
+                                fill 0.3s ease-in-out,
```

Comment:
You may use transition property without prefix
http://caniuse.com/#feat=css-transitions